File: app/src/main/java/wang/julis/jproject/example/media/little/LittleMainActivity.java
package wang.julis.jproject.example.media.little;

import julis.wang.kotlinlearn.RecyclerViewActivity;
import wang.julis.jproject.example.binder.client.BinderTestActivity;
import wang.julis.jwbase.basecompact.baseList.BaseListActivity;

/*******************************************************
 *
 * Created by juliswang on 2022/02/09 14:27 
 *
 * Description : 
 *
 *
 *******************************************************/

public class LittleMainActivity extends BaseListActivity {
    @Override
    protected void initData() {
        addActivity("JsonTest", JsonTestActivity.class);
        addActivity("BinderExample", BinderTestActivity.class);
        addActivity("RecyclerViewActivity", RecyclerViewActivity.class);

    }
}

File: app/src/main/java/wang/julis/jproject/example/little/LittleMainActivity.kt
package wang.julis.jproject.example.little

import wang.julis.jproject.example.binder.client.BinderTestActivity
import wang.julis.jwbase.basecompact.baseList.BaseListActivity

/**
 *
 * Created by @juliswang on 2025/02/14 16:46
 *
 * @Description
 */
class LittleMainActivity : BaseListActivity() {
    override fun initData() {
        addItem("JsonTest", JsonTestActivity::class.java)
        addItem("BinderExample", BinderTestActivity::class.java)
        addItem("文字编码", CharacterDecodingActivity::class.java)
        addItem("Router", RouterActivity::class.java)
        addItem("位标记", BitMarkTest::run)
        addItem("Matrix") { MatrixTest.run(this) }
        addItem("ViewStub", ViewStubActivity::class.java)
    }
}